docs/operational_guide/placement.md

```diff
@@ -0,0 +1,205 @@
+# Placement
+
+## Overview
+
+**Note**: The words *placement* and *topology* are used interchangeably throughout the M3DB documentation and codebase.
+
+A M3DB cluster has exactly one Placement. That placement maps the cluster's shard replicas to hosts. A cluster also has 0 or more namespaces, and each host serves every namespace for the shards it owns. In other words, if the cluster topology states that host A owns shards 1, 2, and 3 then host A will own shards 1, 2, 3 for all configured namespaces in the cluster.
```

Comment:
maybe define a namespace too?

Reply:
I'm just gonna say that its analogous to database tables here, don't want to duplicate definitions everywhere